tag.go

```diff
@@ -88,6 +89,22 @@ func New(name string, opts Options) *Tag {
 		delete(tag.Options, "body")
 	}
 
+	if tag.Options["value"] != nil {
+		val := tag.Options["value"]
+
+		format := tag.Options["time-format"]
```

Comment:
I think we should also allow for a struct tag that specifies the format. that would also make it easier on the decoding side as well if we already know what the format will be.